frontend/src/scenes/data-management/revenue/CurrencyDropdown.tsx

```diff
@@ -0,0 +1,351 @@
+    [SupportedCurrencies.LBP]: '🇱🇧',
+    [SupportedCurrencies.LKR]: '🇱🇰',
+    [SupportedCurrencies.LRD]: '🇱🇷',
+    [SupportedCurrencies.LTL]: '🇱🇹',
+    [SupportedCurrencies.LVL]: '🇱🇻',
+    [SupportedCurrencies.LSL]: '🇱🇸',
+    [SupportedCurrencies.LYD]: '🇱🇾',
+    [SupportedCurrencies.MAD]: '🇲🇦',
+    [SupportedCurrencies.MDL]: '🇲🇩',
+    [SupportedCurrencies.MGA]: '🇲🇬',
+    [SupportedCurrencies.MKD]: '🇲🇰',
+    [SupportedCurrencies.MMK]: '🇲🇲',
+    [SupportedCurrencies.MNT]: '🇲🇳',
+    [SupportedCurrencies.MOP]: '🇲🇴',
+    [SupportedCurrencies.MRU]: '🇲🇷',
+    [SupportedCurrencies.MTL]: '🇲🇹',
+    [SupportedCurrencies.MUR]: '🇲🇺',
+    [SupportedCurrencies.MVR]: '🇲🇻',
+    [SupportedCurrencies.MWK]: '🇲🇼',
+    [SupportedCurrencies.MXN]: '🇲🇽',
+    [SupportedCurrencies.MYR]: '🇲🇾',
+    [SupportedCurrencies.MZN]: '🇲🇿',
+    [SupportedCurrencies.NAD]: '🇳🇦',
+    [SupportedCurrencies.NGN]: '🇳🇬',
+    [SupportedCurrencies.NIO]: '🇳🇮',
+    [SupportedCurrencies.NOK]: '🇳🇴',
+    [SupportedCurrencies.NPR]: '🇳🇵',
+    [SupportedCurrencies.NZD]: '🇳🇿',
+    [SupportedCurrencies.OMR]: '🇴🇲',
+    [SupportedCurrencies.PAB]: '🇵🇦',
+    [SupportedCurrencies.PEN]: '🇵🇪',
+    [SupportedCurrencies.PGK]: '🇵🇬',
+    [SupportedCurrencies.PHP]: '🇵🇭',
+    [SupportedCurrencies.PKR]: '🇵🇰',
+    [SupportedCurrencies.PLN]: '🇵🇱',
+    [SupportedCurrencies.PYG]: '🇵🇾',
+    [SupportedCurrencies.QAR]: '🇶🇦',
+    [SupportedCurrencies.RON]: '🇷🇴',
+    [SupportedCurrencies.RSD]: '🇷🇸',
+    [SupportedCurrencies.RUB]: '🇷🇺',
+    [SupportedCurrencies.RWF]: '🇷🇼',
+    [SupportedCurrencies.SAR]: '🇸🇦',
+    [SupportedCurrencies.SBD]: '🇸🇧',
+    [SupportedCurrencies.SCR]: '🇸🇨',
+    [SupportedCurrencies.SDG]: '🇸🇩',
+    [SupportedCurrencies.SEK]: '🇸🇪',
+    [SupportedCurrencies.SGD]: '🇸🇬',
+    [SupportedCurrencies.SRD]: '🇸🇷',
+    [SupportedCurrencies.SSP]: '🇸🇸',
+    [SupportedCurrencies.STN]: '🇸🇹',
+    [SupportedCurrencies.SYP]: '🇸🇾',
+    [SupportedCurrencies.SZL]: '🇸🇿',
+    [SupportedCurrencies.THB]: '🇹🇭',
+    [SupportedCurrencies.TJS]: '🇹🇯',
+    [SupportedCurrencies.TMT]: '🇹🇲',
+    [SupportedCurrencies.TND]: '🇹🇳',
+    [SupportedCurrencies.TOP]: '🇹🇴',
+    [SupportedCurrencies.TRY]: '🇹🇷',
+    [SupportedCurrencies.TTD]: '🇹🇹',
+    [SupportedCurrencies.TWD]: '🇹🇼',
+    [SupportedCurrencies.TZS]: '🇹🇿',
+    [SupportedCurrencies.UAH]: '🇺🇦',
+    [SupportedCurrencies.UGX]: '🇺🇬',
+    [SupportedCurrencies.USD]: '🇺🇸',
+    [SupportedCurrencies.UYU]: '🇺🇾',
+    [SupportedCurrencies.UZS]: '🇺🇿',
+    [SupportedCurrencies.VES]: '🇻🇪',
+    [SupportedCurrencies.VND]: '🇻🇳',
+    [SupportedCurrencies.VUV]: '🇻🇺',
+    [SupportedCurrencies.WST]: '🇼🇸',
+    [SupportedCurrencies.XAF]: '🌍', // Central Africa currency, using Africa+Europe globe emoji
+    [SupportedCurrencies.XCD]: '🌎', // East Caribbean Dollar, using Americas globe emoji
+    [SupportedCurrencies.XDR]: '🌎', // Special Drawing rights, IMF "currency", Americas globe emoji because I have to pick something
```

Comment:
If this comes up in prod, I will pay you one XDR

Reply:
I could just remove this one right......

Reply:
I removed it anyway 🤣